include/constants/battle_ai.h

```diff
@@ -28,7 +28,7 @@
 #define AI_FLAG_CHECK_BAD_MOVE        (1 << 0)
 #define AI_FLAG_TRY_TO_FAINT          (1 << 1)
 #define AI_FLAG_CHECK_VIABILITY       (1 << 2)
-#define AI_FLAG_SETUP_FIRST_TURN      (1 << 3)
+#define AI_FLAG_FORCE_SETUP_FIRST_TURN      (1 << 3) // AI will prioritize using setup moves on the first turn at the expensve of all else. AI_FLAG_CHECK_VIABILITY will instead do this when the AI determines it makes sense.
 #define AI_FLAG_RISKY                 (1 << 4)
 #define AI_FLAG_PREFER_STRONGEST_MOVE (1 << 5)
 #define AI_FLAG_PREFER_BATON_PASS     (1 << 6)
```

Comment:
would it potentially be useful to document every other flag?

Reply:
Great idea, I gave these all a pass with comments, feel free to suggest alterations to any of them.

This is also probably our best opportunity to change the names of Omniscient and HP Aware if we'd like to, we certainly still can later but those two have been causing some confusion. Not sure if it's necessary right now though if we don't have good ideas for what to use instead.

Reply:
What's confusing about them, and what would be a better name?

Reply:
I don't personally find either confusing, but we've discussed before that HP Aware is really just an arbitrary list of score increase / decreases based on HP rather than making the AI "smart" or "aware", and Omniscient can make it sound like the AI knows what actions the player is taking.

I'm not saying we need to change either, just that this would be a good time to consider whether we'd like to. The primary reason I think we don't is that I at least don't have any better naming suggestions in spite of the confusing, I think the comments describe what they do really well and at some point we have to rely on users reading the details if they care lol

Reply:
I think the comments are good on those for now. We can always think about it in the future